src/main/java/com/spikes2212/command/drivetrains/commands/DriveArcadeWithPID.java

```diff
@@ -5,20 +5,22 @@
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 
-public class DriveWithPID extends CommandBase {
+public class DriveArcadeWithPID extends CommandBase {
 
     private final TankDrivetrain drivetrain;
     private final PIDLoop movementPIDLoop;
+    private double setpoint;
 
     /**
      * @param drivetrain      is the {@link TankDrivetrain} the command moves.
      * @param movementPIDLoop is the {@link PIDLoop} that calculates and sets the speed to the drivetrain.
      */
 
-    public DriveWithPID(TankDrivetrain drivetrain, PIDLoop movementPIDLoop) {
+    public DriveArcadeWithPID(TankDrivetrain drivetrain, PIDLoop movementPIDLoop, double setpoint) {
```

Comment:
make setpoint a `Supplier<Double>` and add a constructor that receives a  `double` value